gensim/corpora/bleicorpus.py

```diff
@@ -108,23 +113,25 @@ def line2doc(self, line):
     def save_corpus(fname, corpus, id2word=None, metadata=False):
         """Save a corpus in the LDA-C format.
 
-        There are actually two files saved: `fname` and `fname.vocab`, where
-        `fname.vocab` is the vocabulary file.
+        Notes
+        -----
+        There are actually two files saved: `fname` and `fname.vocab`, where `fname.vocab` is the vocabulary file.
 
         Parameters
         ----------
         fname : str
-            Filename.
-        corpus : iterable
-            Iterable of documents.
+            Path to output filename.
+        corpus : iterable of iterable of (int, float)
+            Input corpus
```

Comment:
Obvious, no additional information provided. There's no need to have descriptions for _all_ arguments. :)